src/main/java/org/apache/mesos/hdfs/state/AcquisitionPhase.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.mesos.hdfs.state;
+
+public enum AcquisitionPhase {
+  JOURNAL_NODES, NAME_NODE_1, NAME_NODE_2, DATA_NODES
```

Comment:
Please put these on separate lines, and ideally comment what each state/phase means.